GUI/Controls/TextControl.cs

```diff
@@ -5,13 +5,13 @@ namespace GUI.Controls
     partial class TextControl : ControlPanelView
     {
         protected override Panel ControlsPanel => controlsPanel;
-        public CodeTextBox TextBox { get; }
+        public CodeTextBox BetterTextBox { get; }
```

Comment:
Unintended rename. BetterTextBox is a class, the class used here is CodeTextBox.

Reply:
Same in other files.